packages/wrangler/src/proxy.ts

```diff
@@ -484,3 +489,14 @@ export async function waitForPortToBeAvailable(
 		}
 	});
 }
+
+function getAccessibleHosts(): string[] {
+	const hosts: string[] = [];
+	Object.values(networkInterfaces()).forEach((net) => {
+		net?.forEach(({ family, address }) => {
+			// @ts-expect-error the `family` property is numeric as of Node.js 18.0.0
+			if (family === "IPv4" || family === 4) hosts.push(address);
```

Comment:
How come we only do IPv4?

Reply:
I just copied miniflare's implementation to be consistent, happy to change, but then it should also be changed there